stripe/src/main/java/com/stripe/android/paymentsheet/CurrencyFormatter.kt

```diff
@@ -9,19 +9,25 @@ import kotlin.math.pow
 internal class CurrencyFormatter {
     fun format(
         amount: Long,
-        currency: Currency
+        currencyAmount: Currency,
+        targetLocale: Locale = Locale.getDefault()
     ): String {
-        val currencySymbol = currency.getSymbol(Locale.getDefault())
+        val currencySymbol = currencyAmount.getSymbol(targetLocale)
```

Comment:
Is this the underlying issue, or that our Activity isn't reacting to config changes?

Reply:
It's happening when starting the app from the start, so not a config change this time.